modules/microshift-embed-microshift-update-osbuilder-worker.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-offline-deploy.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-embed-microshift-update-osbuilder-worker_{context}"]
+= Updating osbuilder worker configuration to prepare for image building
+
+After you have updated the blueprint, you must update the osbuilder worker configuration to prepare for building the image with embedded {microshift-short} containers.
+
+.Prerequisites
+
+* You have root-user access to your build host.
+* Your build host meets the Image Builder system requirements.
+* You have installed and set up Image Builder and the composer-cli tool.
+* You have created and updated an {op-system-ostree} image blueprint for offline use.
+* You have set up the `/etc/osbuild-worker/osbuild-worker.toml` configuration file.
```

Comment:
These 2 need to be removed.
1. Not related to making the configuration change
2. This is what we're doing in this section